core/predictor/proto/framework.proto

```diff
@@ -0,0 +1,217 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
move this to tools. Plus, I think there is a chance that if the framework desc changed in Paddle repo, it is hard to sync with that file with current method.